python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -338,6 +339,9 @@ def optimize(self, model: nn.Module,
         else:
             sample_size_for_pot = 100
 
+        # patch context manager
+        model.context_manager = BaseContextManager()
```

Comment:
Change to `model._nano_context_manager` to avoid potential conflict?

And we should use something like `with InferenceModel.get_context(...)` or `with InferenceModel.set_context(...)` to ensure interface consistency.

Reply:
Will change to `model._nano_context_manager`.
And we plan to support `with InferenceModel.get_context(...)` in this PR https://github.com/intel-analytics/BigDL/pull/6782